docs/src/main/asciidoc/liquibase.adoc

```diff
@@ -0,0 +1,270 @@
+
+* the Liquibase extension
+* your JDBC driver extension (`quarkus-jdbc-postgresql`, `quarkus-jdbc-h2`, `quarkus-jdbc-mariadb`, ...)
+
+[source,xml]
+--
+<dependencies>
+    <!-- Liquibase specific dependencies -->
+    <dependency>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-liquibase</artifactId>
+    </dependency>
+
+    <!-- JDBC driver dependencies -->
+    <dependency>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-jdbc-postgresql</artifactId>
+    </dependency>
+</dependencies>
+--
+
+Liquibase support relies on the Quarkus datasource config.
+It can be customized for the default datasource as well as for every <<multiple-datasources,named datasource>>.
+First, you need to add the datasource config to the `{config-file}` file
+in order to allow Liquibase to manage the schema.
+Also, you can customize the Liquibase behaviour by using the following properties:
+
+`quarkus.liquibase.migrate-at-start`::
+**true** to execute Liquibase automatically when the application starts, **false** otherwise. +
+**default:** false
+
+`quarkus.liquibase.validate-on-migrate`::
+**true** to validate the applied changes against the available ones, **false** otherwise. It is only used if *migration-at-start* is *true* +
+**default:** true
+
+`quarkus.liquibase.clean-at-start`::
+**true** to execute Liquibase clean command automatically when the application starts, **false** otherwise. +
+**default:** false
+
+`quarkus.liquibase.change-log`::
+The liquibase change log file. All included change log files in this file are scanned and add to the projects. +
+**default:** classpath:db/changeLog.xml
+
+`quarkus.liquibase.database-change-log-table-name`::
+The name of Liquibase's change log table. +
+**default:** DATABASECHANGELOG
+
+`quarkus.liquibase.database-change-log-lock-table-name`::
+The name of Liquibase's lock table. +
+**default:** DATABASECHANGELOGLOCK
+
+`quarkus.liquibase.labels`::
+Comma-separated case-sensitive list of expressions defining labeled ChangeSet to execute for liquibase. +
+**default:** <none>
+
+`quarkus.liquibase.contexts`::
+Comma-separated case-sensitive list of ChangeSet contexts to execute for liquibase. +
+**default:** <none>
+
+`quarkus.liquibase.default-catalog-name`::
+The name of Liquibase's default catalog. +
+**default:** <none>
+
+`quarkus.liquibase.default-schema-name`::
+The name of Liquibase's default schema. Overwrites the default schema name
+(returned by the RDBMS) with a different database schema +
+**default:** <none>
+
+`quarkus.liquibase.liquibase-catalog-name`::
+The name of the catalog with the liquibase tables +
+**default:** <none>
+
+`quarkus.liquibase.liquibase-schema-name`::
+The name of the schema with the liquibase tables. +
+**default:** <none>
+
+`quarkus.liquibase.liquibase-tablespace-name`::
+The name of the tablespace where the -LOG and -LOCK tables will be created (if they do not exist yet) +
+**default:** <none>
```

Comment:
I think this section can be generated automatically if you add 
```adoc
include::{generated-dir}/config/quarkus-liquibase.adoc[opts=optional, leveloffset=+2]
```

Reply:
@gastaldi thanks for the hint. Done